.gitignore

```diff
@@ -19,6 +19,7 @@
 /frontend/packages/integration-tests-cypress/cypress-a11y-report.json
 /frontend/packages/dev-console/integration-tests/cypress-a11y-report.json
 /frontend/packages/ceph-storage-plugin/integration-tests-cypress/cypress-a11y-report.json
+/frontend/packages/kubevirt-plugin/integration-tests-cypress/cypress-a11y-report.json
```

Comment:
I don't think this is needed after #7916.

Reply:
thats right, removed